src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -2636,6 +2641,72 @@ void SurgeGUIEditor::showSettingsMenu(CRect &menuRect)
     eid++;
     mpeSubMenu->forget();
 
+    // Mouse behavior
+
+    int mid = 0;
+
+    COptionMenu* mouseSubMenu =
+        new COptionMenu(menuRect, 0, 0, 0, 0, VSTGUI::COptionMenu::kNoDrawStyle);
+
+    std::string mouseClassic = "Classic";
```

Comment:
Hey cool! Exactly one part of the Surge code is ready for internationalization :) Good stuff here.